### DISCUSSION TITLE
Download of classfd.zip failed

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3277] [General]
Stellarium 23.1.170-f97cbe8 [master] log:
[Satellites] FAILED to download "https://www.prismnet.com/~mmccants/tles/classfd.zip" Error: "Error transferring https://www.prismnet.com/~mmccants/tles/classfd.zip - server replied: Not Found"

The account mmccants has disappeared on PrismNet. That is the reason. This is not a bug of Stellarium.

**Comment by alex-w** [comment]
Thanks for hint!

**Reply by Paul-Gilmartin** [reply]
Try: https://www.mmccants.org/tles/classfd.zip

-- 
gil
